### DISCUSSION TITLE
Action not started on my Pull Request

### DISCUSSION THREAD

**Discussion by 7c00** [community/community#23584] [Pull Requests]
<p>I pushed (with --force) to my pull request. But the actions keeps on the status of ‘Waiting for status to be reported’. What can I do for this?  My pr is <a href="https://github.com/prestodb/presto/pull/17149" class="inline-onebox" rel="noopener nofollow ugc">Add hudi connector by 7c00 · Pull Request #17149 · prestodb/presto · GitHub</a>.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>Right below the statuses it says the PR has a merge conflict. The <a href="https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#pull_request">documentation says</a>:</p>
<blockquote>
<p>Workflows will not run on <code>pull_request</code> activity if the pull request has a merge conflict. The merge conflict must be resolved first.</p>
</blockquote>
